### DISCUSSION TITLE
Setting the font size of ui.step

### DISCUSSION THREAD

**Discussion by frica** [zauberzeug/nicegui#2839] [Q&A] [answered]
### Question

Hello everyone, 

I'm struggling with the styling of my app: I'm trying to change the font size of the ui.step text without altering the font size of the labels below. How can I achieve this?

I must confess the whole Quasar / Tailwind world is new to me, and I'm a bit lost.

Example code:
```py
with ui.stepper().props('vertical').classes('w-full') as stepper:
    with ui.step("Step 1"):
        ui.label("Label first item.")
```

I tried:

```py
with ui.step("Step 1").tailwind.font_size('20px'):
```

But this changes the label below, not the step title.

NB: I use 1.4.20

**Comment by falkoschindler** [comment, marked as answer]
Hi @frica, that's strange. I would have expected your code to work. But there doesn't seem to be an official way to style the title of a [Quasar Step](https://quasar.dev/vue-components/stepper/#qstep-api).

Luckily we can (almost) always add some custom CSS as a workaround:
```py
ui.add_css('.q-stepper__title { font-size: 20px; }')

with ui.stepper().props('vertical').classes('w-full') as stepper:
    with ui.step("Step 1") as step:
        ui.label("Label first item.")
```

**Reply by frica** [reply]
This works indeed. I assume you're overloading the Quasar css?
Thanks a lot @falkoschindler :pray:
